src/material/datepicker/datepicker.md

```diff
@@ -351,7 +351,7 @@ The easiest way to ensure this is to import one of the provided date modules:
 *Please note: `MatNativeDateModule` is based off the functionality available in JavaScript's
 native [`Date` object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Date).
 Thus it is not suitable for many locales. One of the biggest shortcomings of the native `Date`
-object is the inability to set the parse format. We highly recommend using the `MomentDateAdapter`
+object is the inability to set the parse format. You can use the `MomentDateAdapter`
```

Comment:
The thing is we _also_ don't recommend the `NativeDateAdapter`. Should we add something about what we _do_ recommend? (Which I guess currently is to just write your own 😬)

Reply:
Updated the text to strong recommend anything _but_ the native adapter, with moment and _other_ being possible options.